xml/System/Console.xml

```diff
@@ -1924,7 +1924,7 @@
         <Parameter Name="bufferSize" Type="System.Int32" Index="0" FrameworkAlternate="net-5.0;netcore-2.0;netcore-2.1;netcore-2.2;netcore-3.0;netcore-3.1;netframework-1.1;netframework-2.0;netframework-3.0;netframework-3.5;netframework-4.0;netframework-4.5;netframework-4.5.1;netframework-4.5.2;netframework-4.6;netframework-4.6.1;netframework-4.6.2;netframework-4.7;netframework-4.7.1;netframework-4.7.2;netframework-4.8;netstandard-2.0;netstandard-2.1;xamarinandroid-7.1;xamarinios-10.8;xamarinmac-3.0" />
       </Parameters>
       <Docs>
-        <param name="bufferSize">The internal stream buffer size.</param>
+        <param name="bufferSize">The internal stream buffer size. On both .NET Core and .NET Framework, this parameter has no effect.</param>
```

Comment:
@carlossanlop can you provide some feedback on wording here? Is the phrase "On both .NET Core and .NET Framework" redundant perhaps?

Reply:
It is redundant, unless it has an effect on .NET 5.0. @john-h-k is this the case? (.NET 5.0 is not considered a ".NET Core", that's why I ask).

Reply:
@gewarren do we have new guidance on how to group .NET Core (3.1 and older) with .NET 5.0?

Reply:
The wording we used, which is Beth Massi-approved, is ".NET 5 (and .NET Core) and later versions". But I don't think that's what we're looking for here, since this includes .NET Framework (and possibly not .NET 5).

Reply:
So it seems there's no point in indicating the .NET versions at all. Right?